.golangci.yml

```diff
@@ -40,6 +40,7 @@ linters-settings:
   goimports:
     local-prefixes: github.com/signalfx/splunk-otel-go
   revive:
+    confidence: 0.7
```

Comment:
Why `0.7`? Can you leave some comment?
As far as I remember `0.8` is a popular value (default?).

Reply:
There are package comment false positives as well as error capitalization naming false positives. The latter have a confidence level of 0.6, meaning this is the minimum value that prevents those without more.

Reply:
> error capitalization naming false positives.

we can just `//nolint` these

I am afraid that `confidence: 0.7` could hide other potential issues in future.

Reply:
> This would actually be more inclusive.

I thought the `confidence` is working the other way. You are correct, sorry for my misunderstanding.

Reply:
> Prior to this upgrade the default was 0.8

TBH I am not sure what caused the change of the default value. Look like some kind of a bug, but I haven't found any issue by just browsing the code (e.g. https://github.com/golangci/golangci-lint/blob/2f689958c3fbaec353543d6429770c4ffb3a42e6/pkg/golinters/revive.go#L353)...

Reply:
Found it. Here is why the behaviour changed: https://github.com/golangci/golangci-lint/pull/2380